plasmapy/formulary/tests/test_ionization.py

```diff
@@ -3,17 +3,35 @@
 import pytest
 from astropy.constants import c
 
-from plasmapy.formulary.ionization import Z_bal
+from plasmapy.formulary.ionization import Z_bal_, Saha
 
 
-def test_Z_bal():
+def test_Z_bal_():
     n = 1e19 * u.m ** -3
     T_e = 5000 * u.K
 
-    assert (Z_bal(n, T_e) * u.dimensionless_unscaled).unit == u.dimensionless_unscaled
+    assert (Z_bal_(n, T_e) * u.dimensionless_unscaled).unit == u.dimensionless_unscaled
 
     with pytest.warns(u.UnitsWarning):
-        Z_bal(1e19, T_e)
+        Z_bal_(1e19, T_e)
 
     with pytest.raises(u.UnitTypeError):
-        Z_bal(2e19 * u.kg, T_e)
+        Z_bal_(2e19 * u.kg, T_e)
```

Comment:
I haven't done this in awhile, but the tests should test `ionization_balance` instead of `Z_bal_`.  Then you test if `Z_bal_` is called then `ionization_balanced` is called.  I'll see if I can find some of my old code that demonstrates this.